C24_WMDE_Desktop_DE_13

gbirke opened the pull request:
Based on CTRL of Desktop 12, add transaction fee checkbox

Ticket: https://phabricator.wikimedia.org/T379605

gbirke committed:
Prepare campaign C24_WMDE_Desktop_DE_13

Ticket: https://phabricator.wikimedia.org/T379605

gbirke committed:
Prepare CTRL for C24_WMDE_Desktop_DE_13

Reset to CTRL of C24_WMDE_Desktop_DE_12

Ticket: https://phabricator.wikimedia.org/T379605

gbirke committed:
Prepare VAR for C24_WMDE_Desktop_DE_13

Add new event map and style files sunce they will change in the next
commit

Ticket: https://phabricator.wikimedia.org/T379605

gbirke committed:
Implement VAR of C24_WMDE_Desktop_DE_13

- Use form with transaction fee
- Adapt styles, messages and event map to support transaction fee
- Add support for `TransactionFees.scss` in Svingle theme
  - Add color swatches for checkboxes
  - Adapt `TransactionFees` (adding night mode support for Vector)
- Add integration tests for transaction fee. Adapt translator stub to
  support rendering parameters.

Abban reviewed (approved):
Is the VAR banner component supposed to be deleted in the second commit, then re-added in the third?

Abban commented:
Can you rebase main before putting this banner live? The banner will then have the Linux FF line height fix.

moiikana commented:
needs same changes as in https://github.com/wmde/fundraising-banners/pull/619 and a rebase

moiikana committed:
Add timer changes (main rebase)

- also adds missing timer change for VAR12

https://phabricator.wikimedia.org/T378589
